src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -64,7 +64,7 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
         size={48}
         isDisabled={isDisabled}
         onClick={action}
-        ariaLabel={text}
+        ariaLabel={text || tooltip}
```

Comment:
While this does fix the issue, I am wondering - is there any other way we can get `text` to come as a string for the cross button? Only reason is - I am not sure if tooltip is best used here... although in this case it is highly unlikely that tooltip changes.
Just a thought.

Reply:
@adhmenon for Leave meeting button we are getting text as undefined and tooltip value we are getting as "Leave meeting", so when text is undefined it will read the value of tooltip, this will be a short fix.